### PR TITLE
LIVE-1244: Editions header image, caption and article width

### PR DESCRIPTION
**Pull request by faresite**
# Editions header image, caption and article width

### CODE REVIEW

File: src/components/editions/headerImage.tsx [hunk 1]
```diff
@@ -1,38 +1,100 @@
...
 	item: Item;
 }
 
+const sizes: Sizes = {
+	mediaQueries: [
+		{ breakpoint: 'tablet', size: '740px' },
+		{ breakpoint: 'wide', size: '980px' },
+	],
+	default: '100vw',
+};
```
Comment (faresite) [inline src/components/editions/headerImage.tsx:75]: Is this worth moving sizes to editions' `style.ts` ? Seems like something that various components will share.

Comment (JamieB-gu) [reply src/components/editions/headerImage.tsx:75]: If other components include images that match the header image size across all breakpoints then yes.

**Review by joecowton1 — APPROVED** [review]

**Review by abeddow91 — APPROVED** [review]

**Comment by faresite** [comment]
This PR should fix all the merge conflicts, however there is still a broken body border top line that goes across.

File: src/components/editions/article.tsx [hunk 2]
```diff
@@ -62,7 +36,7 @@ const Article: FC<Props> = ({ item }) => {
 		<main>
 			<article>
 				<Header item={item} />
-				<section css={getStyles(item)}>
+				<section css={[bodyStyles, articleStyles]}>
```
Comment (joecowton1) [inline src/components/editions/article.tsx:39]: Can we combine these styles please?

**Review by joecowton1 — APPROVED** [review]

**Pull request merged**